### PR TITLE
6 create basic templates

### LINKED ISSUE #6: create basic templates
- create initial templates for bookle puzzle, puzzle complete, and discussion

### PR DESCRIPTION
**Pull request by sophielambie**

### CODE REVIEW

File: bookle/urls.py
Comment (phantomsheepp) [inline bookle/urls.py:1]: a dash in sign-up and about-us urls?

Comment (phantomsheepp) [inline bookle/urls.py:1]: line 18 spelling - logiv

Comment (phantomsheepp) [inline bookle/urls.py:1]: consistency in case used for names - there's daily_puzzles and edit-account - should probably pick either - or _

**Comment by sophielambie** [comment]
should be good now!!

**Review by phantomsheepp — APPROVED** [review]

**Pull request merged**